### ISSUE TITLE
Filter log by date range

### ISSUE THREAD

**Issue by philipa** [meri-imperiumi/signalk-logbook#40]
Navigation controls would be welcome for browsing larger logs.

As a user, I would like:
- to filter the log view by date range
- to control the date sort order (latest first / earliest first)
- a calendar overview indicating the days on which there are log entries

With an increasingly large log, the _earliest first_ default sort order becomes annoying. You often want to view the latest entries. Perhaps the low hanging fruit is to use something like `react-table` to make the date column sortable. This could also allow filtering by crew member or event category.

**Comment by bergie** [comment]
Yup. The current "last seven days with entries" view is just a hack until I have time to build some filtering UI.

I particularly like the Victron VRM filtering UI, though the first attempt at this in the logbook is likely to be less capable:
<img width="1107" alt="Screenshot 2023-04-11 at 9 11 50" src="https://user-images.githubusercontent.com/3346/231083804-1530d227-9ba9-4ba8-a6aa-3091abecd0be.png">

**Comment by bergie** [comment]
Some initial work for this in #43
